ansible_rulebook/schema/ruleset_schema.json

```diff
@@ -125,8 +124,7 @@
                         {"$ref": "#/$defs/print-event-action"},
                         {"$ref": "#/$defs/debug-action"},
                         {"$ref": "#/$defs/none-action"},
-                        {"$ref": "#/$defs/shutdown-action"},
-                        {"$ref": "#/$defs/echo-action"}
```

Comment:
We should not remove the echo action here if the we don't remove it from the rest of the code. I suggest leaving it for a while, and given the case, remove it completely if a later PR.